python/tk_softimage/menu_generation.py

```diff
@@ -103,7 +103,9 @@ def _add_context_menu(self):
         ctx_name = str(ctx)
 
         # create the sub menu object
-        ctx_menu = self._menu_handle.AddSubMenu(ctx_name)
+        # the label expects a unicode object so we cast it to support when the context may 
+        # contain info with non-ascii character
+        ctx_menu = self._menu_handle.AddSubMenu(ctx_name.decode("utf-8"))
```

Comment:
@manneohrstrom to clarify the last line:
> so maybe makes sense to check if it is a unicode object and only convert in that case?

Did you mean check if it's a *string* object and only convert in that case?